virt/vm_networking/virt-networking-overview.adoc

```diff
@@ -50,9 +50,22 @@ You can connect a VM to an SR-IOV network by performing the following steps:
 
 xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#virt-connecting-vm-to-ovn-secondary-network[Connecting a virtual machine to an OVN-Kubernetes secondary network]::
 
-You can connect a VM to an Open Virtual Network (OVN)-Kubernetes secondary network. To configure an OVN-Kubernetes secondary network and attach a VM to that network, perform the following steps:
+You can connect a VM to an Open Virtual Network (OVN)-Kubernetes secondary network. {VirtProductName} supports the flat layer 2 and localnet topologies for OVN-Kubernetes.
```

Comment:
```suggestion
You can connect a VM to an Open Virtual Network (OVN)-Kubernetes secondary network. {VirtProductName} supports the `layer2` and `localnet` topologies for OVN-Kubernetes.
```

Reply:
See my comment above about the backticks